XIPs/xip-21-transaction-reference-content-type.md

```diff
@@ -35,17 +35,17 @@ The goal of the transaction reference content type is to provide transaction det
 ```ts
 type TransactionReference = {
   /**
-   * The namespace for the networkId
+   * The CAIP-2 chain ID where the transaction happened in the format `<namespace>:<reference>`
    */
-  namespace?: string;
+  chainId?: string;
   /**
    * The networkId for the transaction, in decimal or hexidecimal format
    */
   networkId: number | string;
```

Comment:
The `networkId` is no longer necessary as it's `<reference>` part of the `chainId`.

Reply:
So wonderful to see you here @aurelticot - thank you for your partnership on this! 🤝

cc'ing @neekolas for help with confirming this change to the XIP spec. 🫡

Reply:
Fixed to remove `networkId`. My oversight! Thank you.